pegasus/sites.v3/code.org/views/stats_homepage.haml

```diff
@@ -0,0 +1,21 @@
+- student_accounts_percent = "20%"
+- female_students = "9M"
+- teacher_users = "600K"
+- world_students_hoc_percent = "10%"
+- states_policy_change = "20"
+
```

Comment:
1. We usually use token-substitution so that translators can order a value where it makes sense in their language, and also so they can see full context of the whole sentence.  If we don't do that, it stands to reason we might have challenges.  @tanyaparker or @Hamms might know more.

2. It seems as though some of it can be pulled from a database fairly soon.  Take look at https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/properties.rb for some values we already use.  It will be fine to add things like % if/when you switch over, but it seems likely that it would normally just be a numeric value and you should add commas, percentages, etc.